src/graphql/v2/dataSource.js

```diff
@@ -413,6 +407,22 @@ class FhirDataSource extends DataSource {
         }
         return combinedMeta;
     }
+
+    /**
+     * Concatenate code and/or display
+     * @param {Coding | undefined} sourceMetaTag
+     * @param {Coding | undefined} targetMetaTag
+     * @returns {Object}
+     */
+    updateCombinedMetaTag(sourceMetaTag, targetMetaTag) {
```

Comment:
The param names are reversed here. We need to update the target tag with source tag.